src/DetailsView/components/left-nav/details-view-left-nav.tsx

```diff
@@ -67,11 +67,7 @@ export const DetailsViewLeftNav = NamedFC<DetailsViewLeftNavProps>('DetailsViewL
 
     const leftNav: JSX.Element = (
         <div className={`${styles.leftNav} main-nav`}>
-            <Switcher
-                deps={props.deps}
-                featureFlagStoreData={featureFlagStoreData}
-                pivotKey={props.selectedPivot}
-            />
+            <Switcher deps={props.deps} pivotKey={props.selectedPivot} />
```

Comment:
hope we tested after removing featureFlagStoreData

Reply:
Yes, tested and featureFlagStoreData property was removed from its class Switcher which extends SwitcherProps in its previous commit.